core/src/main/java/com/alibaba/fescar/core/constants/ConfigurationKeys.java

```diff
@@ -32,6 +32,22 @@
      * The constant STORE_PREFIX.
      */
     public static final String STORE_PREFIX = "store.";
+
+    /**
+     * The constant STORE_MODE.
+     */
+    public static final String STORE_MODE =  STORE_PREFIX + "mode";
+
+    /**
+     * The constant STORE_FILE_PREFIX
+     */
+    public static final String STORE_FILE_PREFIX = "file.";
```

Comment:
lack DB_PREFIX

Reply:
Maybe STORE_FILE_PREFIX = STORE_PREFIX + "file."?
not STORE_FILE_PREFIX = "file."

Reply:
@CoffeeLatte007   done, exclude DB_PREFIX which is in another PR after 0.5.0